src/main/java/mate/academy/controllers/ShoppingCartController.java

```diff
@@ -12,15 +12,15 @@
 import mate.academy.service.ShoppingCartService;
 
 public class ShoppingCartController extends HttpServlet {
-    private static final Long USER_ID = 1L;
     private static final Injector INJECTOR = Injector.getInstance("mate.academy");
     private ShoppingCartService shoppingCartService =
             (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        Long userId = (Long) req.getSession().getAttribute("user_id");
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
         List<Product> productList = shoppingCart.getProducts();
         req.setAttribute("products", productList);
```

Comment:
```suggestion
        req.setAttribute("products", shoppingCart.getProducts());
```
давай трішки скоротимо код